kafka-ui-react-app/src/components/Wizard/WizardForm/KafkaCluster/KafkaCluster.tsx

```diff
@@ -54,7 +54,7 @@ const KafkaCluster: React.FC = () => {
         <InputHint>
           the list of Kafka brokers that you want to connect to
         </InputHint>
-        <S.BootstrapServersContainer>
+        <S.Container>
```

Comment:
Container of what? I would suggest something like `ArrayFieldWrapper`

Reply:
ok